package.json

```diff
@@ -16,6 +16,7 @@
   ],
   "scripts": {
     "start": "node dist/server.js",
+    "start:dev": "nodemon",
```

Comment:
I've just noticed that `nodemon` isn't included in the dev dependencies, probably something we should add to avoid additional steps.

Reply:
![](https://thumbs.gfycat.com/CrispFirstAurochs-size_restricted.gif)